src/meta/meta_bulk_load_service.cpp

```diff
@@ -40,6 +40,12 @@ DSN_DEFINE_bool("meta_server",
                 "verify files according to metadata before ingest");
 DSN_TAG_VARIABLE(bulk_load_verify_before_ingest, FT_MUTABLE);
 
+DSN_DEFINE_bool("meta_server",
+                concurrent_bulk_load,
+                false,
+                "different app can execute bulk load at the same time if this value is true");
```

Comment:
How about:
'whether to enable different apps to execute bulk load at the same time' ?